dcos/cluster.py

```diff
@@ -381,7 +383,7 @@ def get_dcos_version(self):
             resp = requests.request(
                 'GET',
                 url=endpoint,
-                timeout=http.DEFAULT_TIMEOUT,
+                timeout=http.FAIL_FAST_TIMEOUT,
```

Comment:
This changes the default timeout for this operation from 5 seconds to 3. Is that desired?